pegasus/migrations/109_add_hoc_learn_activity_table.rb

```diff
@@ -0,0 +1,11 @@
+Sequel.migration do
+  change do
+    create_table(:hoc_learn_activity, charset: 'utf8') do
+      primary_key :id
+      String :session, index: true, size: 50
+      String :referer, size: 50
+      String :tutorial, index: true, null: false, size: 50
+      DateTime :created_at, index: true
```

Comment:
Good point. Do we want an `updated_at` column? Do we want timestamps to follow `:id`? Note that later migrations can't specify column order, so will awkwardly sandwich timestamps if they appear at the end.